app/src/main/res/drawable/ic_update_image.xml

```diff
@@ -0,0 +1,366 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
codestyle or xml problem? We need to check because I already saw this with Joao and we dont have the bug anymore. Maybe we will have to check this with the team